prober/derp.go

```diff
@@ -431,6 +431,7 @@ func runDerpProbeQueuingDelayContinously(ctx context.Context, from, to *tailcfg.
 		t := time.NewTicker(time.Second / time.Duration(packetsPerSecond))
 		defer t.Stop()
 
+		toPubDERPKey := toc.SelfPublicKey()
```

Comment:
Nit: just noticed that the format of the to / from key names are slightly different (`toPubDERPKey` vs `fromDERPPubKey`), would be good to have these be the same format (e.g., `(to|from)DERPPubKey`)